csharp/ql/src/semmle/code/csharp/controlflow/ControlFlowGraph.qll

```diff
@@ -1774,6 +1775,11 @@ module ControlFlow {
         private class ExitingCall extends NonReturningCall {
           ExitingCall() {
             this.getTarget() instanceof ExitingCallable
+            or
+            exists(AssertMethod m |
+              m = this.(FailingAssertion).getAssertMethod() |
+              not exists(m.getExceptionClass())
```

Comment:
Are there actually any Assert methods that do terminate the process when they fail?

Reply:
Debug.Assert